libraries/ui/src/default-config/tailwind.css

```
@@ -319,71 +324,73 @@
 }
 
 @font-face {
-    font-family: "Reckless Neue";
-    src: url("https://bluedot.org/fonts/RecklessNeue-Light.woff2") format("woff2");
-    font-weight: 300;
+    font-family: "Inter";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)